enhancements/opev-0015.md

```diff
@@ -0,0 +1,121 @@
+# OPEV #0015: Expansion of the Vulnerability Field
+
+During the development of the initial implementations, there have been issues
+raised about the need to expand the vulnerability field. While this OPEV only
+addresses the need to expand the vulnerability field 1) become an object and to
+2) to express more names or aliases of a vulnerability, members of the community
+[have pointed to other cases](https://github.com/openvex/spec/issues/31) where
+the vulnerability entry needs to be expanded from an identifier string to a full
+struct.
+
+## Issues Addressed by thie OPEV:
+
+This proposal addesses a breaking change to allow for current proposals and
+allow room for growth for future enhancements. Specifically, this proposal 
+provides a fix for the following issue:
+
+* [Allow for vulnerability to be a list](https://github.com/openvex/spec/issues/12)
+
+## Enhancement Proposal
+
+The current entry for a vulnerability in a VEX statement is only a string
+capturing the identifier string. This proposal suggests expanding the 
+vulnerability entry to become an object and allow a list of aliases. 
+
+```json
+{
+  "@context": "https://openvex.dev/ns",
+  "@id": "https://openvex.dev/docs/public/vex-6e8cc8c4d0c38e9e9a60949e5dc0279684d8ad5f6711d9a12bb52247b6cc7271",
+  "author": "mailto:john@doe.com",,
+  "timestamp": "2023-01-16T20:41:55.708329108-06:00",
+  "statements": [
+    {
+      "vulnerability": {
+        "@id": "https://nvd.nist.gov/vuln/detail/CVE-2019-17571",
+        "name": "CVE-2019-17571", 
+        "aliases": [
+            "CVE-2019-17571",
```

Comment:
Is the duplication of the "name" value intentional? If so, that may need to be called out in the definition of aliases.

Reply:
I don't think the duplicate ids would be an error but it is misleading to have it twice in the example. I've removed it.

Reply:
I've made it explicit in 3a059d00b75155c245b208be9394cf1a363fcdcc that the identifier in `name` can be repeated in `aliases`.